integreat_compass/cms/models/organizations/contact.py

```diff
@@ -11,16 +11,16 @@ class Contact(AbstractBaseModel):
     """
 
     creator = models.ForeignKey(User, on_delete=models.CASCADE, null=False)
-    name = models.CharField(
+    contact_name = models.CharField(
```

Comment:
Why these renamings? It is unnecessary to prefix all attributes with the name of the class. Please undo.

Reply:
See above